rholang/src/main/scala/coop/rchain/rholang/interpreter/matcher/package.scala

```diff
@@ -80,23 +80,11 @@ package object matcher {
       private def runFreeMapAndStream: ErroredOrCost[Stream[(FreeMap, A)]] =
         StreamT.run(s.run(Map.empty))
 
-      def takeFirst(): NonDetFreeMapWithCost[A] =
```

Comment:
You can remove this because this gets derived with `implicit def splittableInd`?

Reply:
Yup! I've just double-checked with compiler-san and he agrees OwO